### PR TITLE
LPD-31557 | master

### PR DESCRIPTION
**Pull request by fmarcio**
Jira tickets:
https://liferay.atlassian.net/browse/LPD-31557
https://liferay.atlassian.net/browse/LPD-31233
https://liferay.atlassian.net/browse/LPD-31234

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:114]: Hey @fmarcio, I think we already have a function to do this

```suggestion
		await navigateTo({
			page,
			pageName: 'All Search Terms',
		});
```

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:95]: Hey @fmarcio, I think it is not necessary. The keywords of the search term card are not related to the session. Could you try without this part?

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:80]: Hey @fmarcio, I think you don't need to create this variable (`individualsEvent`), since it is not being reused in another step. What do you think?

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:78]: Hey @fmarcio, I think you don't need to create this variable (`individualsEvent`), since it is not being reused in another step. What do you think?

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:46]: Hey @fmarcio, I think you don't need to create this variable (`individualName`), since it is not being reused in another step. What do you think?

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:48]: Hey @fmarcio, I think you can move this part to be used inside the `Create Individuals Events` step. What do you think?

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:37]: Hey @fmarcio, We are adding this `tag` to each automation to represent the ticket (Story, bug) that originated this automation. Could you add this?

```suggestion
test('Verify that clicking the "All Search Terms" link redirects the user to the Search Terms tab with the same time filter previously set.', 
        {
		tag: '@LPD-26181',
	},
	async ({apiHelpers, page}) => {
```

**Comment by fmarcio** [comment]
It is blocked by this PR [liferay-frontend#4340](https://github.com/liferay-frontend/liferay-portal/pull/4340). Do not move forward until it be merged.

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts]: done

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts]: done

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts]: done

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts]: done

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts]: done

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts]: done

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts:36]: Hey @fmarcio, We can put the ID of the Story or BUG that originated this automation.
```suggestion
		tag: '@LPD-26181',
```

**Review by ViniLopes20 — APPROVED** [review]
Very good, @fmarcio. Thanks a lot 🎉

File: modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts
Comment (fmarcio) [reply modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts]: done

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
Lindoooo, could you rebase your PR with Brian's master? You need to do that [after this merge](https://github.com/brianchandotcom/liferay-portal/pull/152645), but before, just wait for the ci stable and then send it directly

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:analytics-cloud-acceptance

**Comment by fmarcio** [comment]
ci:forward

**Comment by fmarcio** [comment]
ci:stop

**Comment by fmarcio** [comment]
ci:stop

**Pull request closed without merging**
